index.html

```diff
@@ -8,12 +8,124 @@
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+  <div class="container">
+
+    <div class="nav-bar">
+    
+      <div>
+    
+        <a class="logo-link" href="index.html">
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </a>
```

Comment:
This is a very nice addition to your code, I like having your logo image turned into a link just makes the whole project that much better.